packages/main/src/Button.ts

```diff
@@ -364,12 +354,11 @@ class Button extends UI5Element implements IFormElement, IButton {
 		}
 
 		markEvent(e, "button");
-		const formSupport = getFeature<typeof FormSupport>("FormSupport");
-		if (formSupport && this._isSubmit) {
-			formSupport.triggerFormSubmit(this);
+		if (this.internals_?.form && this._isSubmit) {
```

Comment:
submitForm and resetForm internally check for  this.internals_?.form, we can simplify the condition a little bit.